server/tserver/src/main/java/org/apache/accumulo/tserver/tablet/Tablet.java

```diff
@@ -585,15 +583,17 @@ private MinorCompactionTask createMinorCompactionTask(long flushId,
       MinorCompactionReason mincReason) {
     MinorCompactionTask mct;
     long t1, t2;
-
+    long pauseLimit =
+        getContext().getTableConfiguration(extent.tableId()).getCount(Property.TABLE_FILE_PAUSE);
     StringBuilder logMessage = null;
 
     try {
       synchronized (this) {
         t1 = System.currentTimeMillis();
 
         if (isClosing() || isClosed() || getTabletMemory().memoryReservedForMinC()
-            || getTabletMemory().getMemTable().getNumEntries() == 0 || updatingFlushID) {
```

Comment:
Should not remove the ` getTabletMemory().getMemTable().getNumEntries() == 0` check.  We do not want to bother initiating a minor compaction for a tablet that has not key/values in memory.